core/extension/gdnative_interface.cpp

```diff
@@ -661,6 +661,20 @@ static const char32_t *gdnative_string_operator_index_const(const GDNativeString
 	return &self->ptr()[p_index];
 }
 
+/* Packed array functions */
+
+static uint8_t *gdnative_packed_byte_array_operator_index(GDNativeTypePtr p_self, GDNativeInt p_index) {
+	PackedByteArray *self = (PackedByteArray *)p_self;
+	ERR_FAIL_INDEX_V(p_index, self->size() + 1, nullptr);
```

Comment:
This should be only `self->size()`, without the `+ 1`. Same for the other function. Otherwise looks good.

```suggestion
	ERR_FAIL_INDEX_V(p_index, self->size(), nullptr);
```

Reply:
oh, offcourse the +1 is from the original function having the zero terminator